lib/src/facts/bsd/networking_resolver.cc

```diff
@@ -119,10 +119,20 @@ namespace facter { namespace facts { namespace bsd {
 
     string networking_resolver::get_primary_interface() const
     {
-        // By default, use the fallback logic of looking for the first interface
-        // that has a non-loopback address
-        return {};
+        string interface;
+        each_line("route", { "-n", "get",  "default" }, [&interface](string& line){
```

Comment:
You may consider using the full path here (`/sbin/route`) to make the fact available even if `/sbin` is not in the PATH.

Reply:
Oh, it looks like when `/sbin` is not in the PATH, facter still finds the route command path.  You can ignore the above comment.